### PR TITLE
FIX

### PR DESCRIPTION
**Pull request by richardshaju**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -25,7 +25,8 @@ import { LuFilter } from "react-icons/lu";
 import NoEvents from "./NoEvents";
 import FCM from "@/components/ui/fcm";
 import moment from "moment";
-import CountdownTimer from "@/components/ui/countdown";
+import CountDown from "@/components/ui/CountDown";
```
Comment (chethaslp) [inline app/page.tsx:28]: Fix this import

File name's wrong.

**Review by chethaslp — CHANGES_REQUESTED** [review]

File: app/page.tsx (hunk 1 above)
Comment (richardshaju) [reply app/page.tsx:28]: Actually, I had changed the file name but it's not showing in the commit

**Comment by chethaslp** [comment]
Failed to compile.
./app/page.tsx
Module not found: Can't resolve '@/components/ui/CountDown'
https://nextjs.org/docs/messages/module-not-found

**Comment by richardshaju** [comment]
@chethaslp Resolved

**Pull request merged**